fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/LogicalCompatibilityContext.java

```diff
@@ -92,31 +92,28 @@ public BiMap<Integer, Integer> getQueryToViewNodeIDMapping() {
         return queryToViewNodeIDMapping;
     }
 
-    /**
-     * Get all expression mapping in query to view
-     */
-    @Deprecated
-    public BiMap<Expression, Expression> getQueryToViewAllExpressionMapping() {
-        if (queryToViewAllExpressionMapping != null) {
-            return queryToViewAllExpressionMapping;
-        }
-        queryToViewAllExpressionMapping = HashBiMap.create();
-        queryToViewAllExpressionMapping.putAll(getQueryToViewJoinEdgeExpressionMapping());
-        queryToViewAllExpressionMapping.putAll(getQueryToViewNodeExpressionMapping());
-        queryToViewAllExpressionMapping.putAll(getQueryToViewFilterEdgeExpressionMapping());
-        return queryToViewAllExpressionMapping;
+    public Expression getQueryJoinExprFromView(Expression viewJoinExpr) {
+        return queryToViewJoinEdgeExpressionMappingSupplier.get().inverse().get(viewJoinExpr);
```

Comment:
`inverse()` need calc, Maybe we should construct `viewToQueryJoinEdgeExpressionMapping`  field in context for performance